front/pages/w/[wId]/developers/api-keys.tsx

```diff
@@ -362,14 +365,24 @@ export function APIKeys({
                             </p>
                           )}
                           <pre className="text-sm">{key.secret}</pre>
-                          <p className="front-normal text-xs text-element-700">
+                          <p
```

Comment:
`nit`: all those could probably be `ContextItem` no? (just so that we specify the dark classes there

Reply:
Yes ideally we rarely need specify classes on front, we can definitely migrate to Sparkle (won't do in this PR but happy to do later)